spec/lib/expired_record_cleaner_spec.rb

```diff
@@ -0,0 +1,33 @@
+describe ExpiredRecordCleaner do
+  describe "fiscal year" do
```

Comment:
since the only method we are using in the Rake task is `vacuum_old_proposals`, what do you think about just writing a test for that and making these other methods private? I think we could still test which proposals are deleted etc within the context of that spec. 

The reason I think it would be helpful to have the other methods be private is that having them private would indicate that they are used within the class only.

Also, while we are testing `vacuum_old_proposals`, what do you think about adding an expectation around the mailer it sends out? I almost missed that part until I saw the mailer spec.